backend/src/server/routes/v1/pki-sync-routers/pki-sync-router.ts

```diff
@@ -76,6 +78,27 @@ const PkiSyncOptionsSchema = z.object({
   minCertificateNameLength: z.number().optional()
 });
 
+const PkiSyncCertificateSchema = z.object({
+  id: z.string().uuid(),
+  pkiSyncId: z.string().uuid(),
+  certificateId: z.string().uuid(),
+  syncStatus: z.nativeEnum(CertificateSyncStatus),
+  lastSyncMessage: z.string().nullable().optional(),
+  lastSyncedAt: z.date().nullable().optional(),
+  createdAt: z.date(),
+  updatedAt: z.date(),
+  certificateSerialNumber: z.string().optional(),
```

Comment:
IGNORE PLACEMENT OF COMMENT

Minor but "Aws" here should be capitalized I think
<img width="1165" height="545" alt="image" src="https://github.com/user-attachments/assets/b92c6a33-3be9-4c42-b2b8-373533de8e0d" />